docs/proposals/migration/design-of-seamless-cluster-migration-scheme.md

```diff
@@ -0,0 +1,310 @@
+---
+title: proposal of seamless cluster migration
+authors:
+- "@chaosi-zju"
+reviewers:
+- "@RainbowMango"
+- TBD
+approvers:
+- "@RainbowMango"
+- TBD
```

Comment:
Please pick reviewers from this PR conversation and put them on the reviewer list.
Then, remove TBD.